src/libs/ReportActionsUtils.ts

```diff
@@ -371,7 +372,22 @@ function replaceBaseURL(reportAction: ReportAction): ReportAction {
     if (!updatedReportAction.message) {
         return updatedReportAction;
     }
-    updatedReportAction.message[0].html = reportAction.message[0].html.replace('%baseURL', environmentURL);
+    const reportID = reportAction.originalMessage.reportID;
+    const roomName = allReports ? allReports[`${reportID}`]?.reportName : '';
+    const verb = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG.INVITE_TO_ROOM ? 'invited' : 'removed';
+    const listMention = (reportAction.originalMessage.targetAccountIDs ?? []).map((accountID) => `<mention-user accountID=${accountID}></mention-user>`);
+
+    const lastMention = listMention.pop();
+    let lastPrefix = ', and ';
+    if (listMention.length === 0) {
+        lastPrefix = '';
+    }
+    if (listMention.length === 1) {
+        lastPrefix = ' and ';
+    }
+    const linkToRoom = `<a href=${environmentURL}/r/${reportID}>${roomName}</a>`;
+    updatedReportAction.message[0].html = `<muted-text>${verb} ${listMention.join(', ')}${lastPrefix}${lastMention} ${linkToRoom}</muted-text>`;
+    updatedReportAction.originalMessage.roomName = roomName;
```

Comment:
Yeah, I like that idea too. Let me cc @jasperhuangg to see if he has any thoughts since he worked on these logics.

Reply:
I am working on the backend change and have a quick question to understand the behavior here. Let's say we update the name of a room, so would that trigger a call to this function for #admin room? Otherwise, wouldn't that require a reload for the #admin room to show the updated name correctly?

Reply:
If I add a placeholder string for roomName in backend then this functionality would break so can we ship this change with OR condition to handle both cases and once it has been live for a few days we can get rid of OR part and just rely on the placeholder part?

Reply:
Ok, yeah I agree we can move forward with PR and once the BE is ready we can refactor it.

Reply:
Hey, can anyone answer this question for me?

> Let's say we update the name of a user-created room, so would that trigger a call to this function for #admin room? Otherwise, wouldn't that require a reload for the #admin room to show the updated name correctly?